fMain.cs

```diff
@@ -888,7 +880,12 @@ private void Randomize(BackgroundWorker worker, DoWorkEventArgs e)
             //Sort BGM
             SeedRNG();
             SortBGM();
-        }
+        }
+
+        private void tSeed_TextChanged(object sender, EventArgs e)
+        {

```

Comment:
Is this used? If the seed is saved during `tSeed_Enter` and `tSeed_Leave`, this empty function serves no purpose but noise.

Reply:
I think this was automatically generated by the designer at some point. I'll remove it.